.cirrus.yml

```diff
@@ -18,7 +18,9 @@ unittest_task:
   << : *cargo_task_template
   container:
     image: rust:bookworm
-  build_script: make V=1 CARGO_OPTS=--verbose
+  build_script:
+    - cargo install cargo-llvm-cov
```

Comment:
this is not currently used in the CI, so I guess the hunks on this file can be dropped.